assets/queries/openAPI/2.0/schema_with_additional_properties_set_as_boolean/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "3a01790c-ebee-4da6-8fd3-e78657383b75",
+  "queryName": "Schema with 'additionalProperties' set as Boolean",
+  "severity": "INFO",
+  "category": "Best Practices",
+  "descriptionText": "The value of 'additionalProperties' should be set as object instead of boolean since openAPI 2.0 does not support boolean value for it",
```

Comment:
```suggestion
  "descriptionText": "The value of 'additionalProperties' should be set as schema object instead of boolean, since swagger 2.0 does not support boolean value for it",
```